src/safeposix/syscalls/net_calls.rs

```diff
@@ -244,18 +310,31 @@ impl Cage {
                 let filename = truepath.file_name().unwrap().to_str().unwrap().to_string(); //for now we assume this is sane, but maybe this should be checked later
 
                 //this may end up skipping an inode number in the case of ENOTDIR, but that's not catastrophic
+                //FS_METADATA contains information about the file system
                 let newinodenum = FS_METADATA
                     .nextinode
-                    .fetch_add(1, interface::RustAtomicOrdering::Relaxed); //fetch_add returns the previous value, which is the inode number we want
+                    .fetch_add(1, interface::RustAtomicOrdering::Relaxed); 
+                    //fetch_add returns the previous value, which is the inode number we want,
```

Comment:
this is good for your understanding but im not sure this needs a lenghty comment, but ill defer to Justin

Reply:
I'd say the comment can be shortened, but something is needed here.  I'm actually more interested in *why* relaxed ordering was used instead of what relaxed ordering does...

Reply:
We just went with the fastest ordering guaranteed to be atomic since we probably don't need to be more restrictive for a counter.

https://stackoverflow.com/questions/30407121/which-stdsyncatomicordering-to-use